scintilla/lexers/LexCPP.cxx

```diff
@@ -1327,10 +1327,13 @@ void SCI_METHOD LexerCPP::Fold(unsigned int startPos, int length, int initStyle,
 			} else {
 				if ((ch == '/') && (chNext == '/')) {
 					char chNext2 = styler.SafeGetCharAt(i + 2);
-					if (chNext2 == '{') {
-						levelNext++;
-					} else if (chNext2 == '}') {
-						levelNext--;
+					if (!inLineComment) { //bracket not in singleline comment
+						if (chNext2 == '{') {
+							levelNext++;
+						}
+						else if (chNext2 == '}') {

```

Comment:
Don't change coding style (as stated in [Guideline](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#guidelines-for-pull-requests)) i.e. write lines 1333 and 1334 as: `} else if (chNext2 == '}') {`.